app/plugin_install.go

```diff
@@ -91,6 +91,14 @@ func (a *App) installPlugin(pluginFile io.Reader, replace bool) (*model.Manifest
 		mlog.Error("failed to notify plugin status changed", mlog.Err(err))
 	}
 
+	// Store bundle in the file store to allow access from other servers.
```

Comment:
Propose moving this above the `EnablePlugin` on line 87 above, otherwise other servers will likely see the event and try to fetch the plugin, but this `WriteFile` might not have succeeded.

Reply:
Yes, good catch 👍